test/test_no_multiply_numbers.py

```diff
@@ -0,0 +1,409 @@
+# type: ignore
+"""
+
+    test_no_multiply_numbers.py
+
+    Tests for Greynir no_multiply_numbers flag functionality
+
+    Copyright(C) 2021 by Miðeind ehf.
```

Comment:
Modify copyright year to 2022, and the Original author can be skipped.

Reply:
Fixed!